result/bmpprefix.yaml

```diff
@@ -0,0 +1,652 @@
+components:
+  schemas:
+    BmpPrefix.State.Request:
+      description: >-
+        The request to retrieve BMP server learned BGP peer prefix information.
+      type: object
+      properties:
+        peer_bgp_id:
+          description: >-
+            The identifiers of BGP peers for which prefix information will be
+            retrieved. If no names are specified then the results will
+            contain prefix information for all BGP peers in BMP server database.
+          type: array
+          items:
+            type: string
+          x-field-uid: 1
+        prefix_filters:
```

Comment:
I see it is there in BGP but confusing and best avoided I feel ( from legacy initial modeling ; best avoided I think )